index.d.ts

```diff
@@ -3,12 +3,14 @@ declare module 'react-native-contacts' {
   export function getAllWithoutPhotos(callback: (error: any, contacts: Contact[]) => void): void;
   export function getPhotoForId(contactId: string, callback: (error: any, photoUri: string) => void): void;
   export function addContact(contact: Contact, callback: (error?: any) => void): void;
-  export function openContactForm(contact: Contact, callback: (error?: any) => void): void;
+  export function openContactForm(contact: Contact, callback: (error: any, contact: Contact) => void): void;
+  export function openExistingContact(contact: Contact, callback: (error: any, contact: Contact) => void): void;
   export function updateContact(contact: Contact, callback: (error?: any) => void): void;
   export function deleteContact(contact: Contact, callback: (error?: any) => void): void;
   export function getContactsMatchingString(str: string, callback: (error: any, contacts: Contact[]) => void): void;
   export function checkPermission(callback: (error: any, result: 'authorized' | 'denied' | 'undefined') => void): void;
   export function requestPermission(callback: (error: any, result: 'authorized' | 'denied' | 'undefined') => void): void;
+  export function writePhotoToPath(contactId: string, file: string, callback: (error: any, result: boolean) => void): void;
```

Comment:
This method has no implementation on iOS. Write an implementation that will just return an error ('Error method not implemented on iOS')